pkgs/tools/cd-dvd/uif2iso/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, fetchurl, unzip, zlib }:
+
+stdenv.mkDerivation rec {
+  nameNoVer = "uif2iso";
+  name = "${nameNoVer}-0.1.7";
+
+  src = fetchurl {
+    url = "http://aluigi.altervista.org/mytoolz/${nameNoVer}.zip";
+    sha256 = "1v18fmlzhkkhv8xdc9dyvl8vamwg3ka4dsrg7vvmk1f2iczdx3dp";
+  };
+
+  buildInputs = [unzip zlib];
+
+  unpackPhase = ''
+    unzip $src
```

Comment:
I've removed the buildPhase and unlockPhase.
However I can't remove the installPhase, as otherwise the executable is in usr/bin rather than bin, and the executable isn't found unless nix profile's usr/bin is added to the path as well.